src/platform/forms-system/test/js/review/SubmitButtons.unit.spec.jsx

```diff
@@ -61,7 +61,7 @@ describe('Schemaform review: <SubmitButtons>', () => {
       <SubmitButtons submission={submission} formConfig={formConfig} />,
     );
 
-    expect(tree.everySubTree('.usa-alert-error')).not.to.be.empty;
+    expect(tree.everySubTree('ErrorMessage')).not.to.be.empty;
```

Comment:
We are using `skin-deep` in this test. What are our thoughts on updating this to RTL in this iteration vs a future iteration?

Reply:
I would say go ahead and update the single test to RTL, and leave SkinDeep in all the others to be taken care of in the future.

Reply:
My preference would be to continue using SkinDeep on all of these until we convert them all over to RTL. Mostly it's just a hunch that if we use more than one tool to accomplish the same job in one module, bad things are likely to happen. It's not a strong opinion, though.